Imagine/Filter/PostProcessor/OptiPngPostProcessor.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Liip\ImagineBundle\Imagine\Filter\PostProcessor;
+
+use Liip\ImagineBundle\Binary\BinaryInterface;
+use Liip\ImagineBundle\Model\Binary;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\ProcessBuilder;
+
+class OptiPngPostProcessor implements PostProcessorInterface
+{
+    const MIN_LEVEL = 0;
+    const MAX_LEVEL = 7;
+
+    private static $allowedTypes = array(
```

Comment:
I think that this array could be used as default but there is a way to extend it, what about second constructor argument?